components/schema-migrator/migrations/director/20230209151200_introduce-managed-application-consumer.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE cert_subject_mapping
+    DROP CONSTRAINT cert_subject_mapping_consumer_type_check;
+
+COMMIT;
```

Comment:
This could be removed because it will be adapted in https://github.com/kyma-incubator/compass/pull/2878
The down migration as well